test/e2e/flask/multichain-api/wallet_createSession.spec.ts

```diff
@@ -387,4 +387,112 @@ describe('Multichain API', function () {
       });
     });
   });
+
+  describe(`Dapp has existing session with scopes eip155:1337, eip155:1 & eip155:42161, and accounts ${ACCOUNT_1} & ${ACCOUNT_2}, call 'wallet_createSession' with scopes eip:155:1338 & eip:155:1000, and account 0xf68464152d7289d7ea9a2bec2e0035c45188223c`, function () {
```

Comment:
```suggestion
  describe(`Dapp has existing session with 3 scopes and 2 accounts and then calls 'wallet_createSession' with different scopes and accounts`, function () {
```

or something like that? It seems a bit too specific of a scenario description otherwise

Reply:
resolved in [ebd7994](https://github.com/MetaMask/metamask-extension/pull/29672/commits/ebd7994059ea3e75ef7481f24c8fe593d46a12a2)